includes/controls/groups/flex-container.php

```diff
@@ -167,7 +167,7 @@ protected function init_fields() {
 					'max' => 100,
 				],
 			],
-			'size_units' => [ 'px', '%', 'vw' ],
+			'size_units' => [ 'px', '%', 'vw', 'em' ],
 			'selectors' => [
```

Comment:
Can we add a range e.g. from 0 to 50?

Reply:
@avivu 

All unit sizes have range. I would expect a range for 'em' as well. What do you think?

Reply:
Done